packages/concatjs/_README.md

```diff
@@ -13,14 +13,28 @@ around each file so that the Chrome DevTools shows the files in the tree as if t
 However at its core, concatjs requires a big tradeoff of a migration cost to buy-in, to get this incredible performance.
 The path of the JavaScript files is lost in the bundling process, so they must contain their module ID internally.
 
-Named AMD/UMD (non-anonymous require.js modules) and `goog.module` are the two JS module formats that are compatible with concatjs.
-Most packages do not ship with this format, so in order to use concatjs tooling, you have to shim your code and dependencies.
+[Named AMD/UMD modules](https://requirejs.org/docs/whyamd.html#namedmodules) and `goog.module` are the two JS module formats that are compatible with concatjs.
+Most packages do not ship with this format, so in order to use concatjs tooling, you have to shim your code and dependencies. See the [Compatibility](#compatibility) section below.
 
 This is at the core of how Google does JavaScript development.
 So Bazel rules that originated in Google's codebase have affordances for concatjs.
 For example `ts_library` produces named AMD modules in its "devmode" output, and
 `karma_web_test` expects to bundle inputs using concatjs.
 
+## Compatibility
+
+To make it easier to produce a UMD version of a third-party npm package, we automatically generate a target that uses Browserify to build one, using the `main` entry from the package's `package.json`.
+In most cases this will make the package loadable under concatjs.
+This target has a `__umd` suffix. For example, if your library is at `@npm//foo` then the UMD target is `@npm//foo:foo__umd`.
+
+An example where this fixes a users issue: <https://github.com/bazelbuild/rules_nodejs/issues/2317#issuecomment-735921318>
+
+In some cases, the generated UMD bundle is not sufficient.
```

Comment:
Also in some cases the generated target will fail to build as we use a fairly generic browserify config.

Reply:
good callout, adding